utils/tmux_utils.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+get_tmux_option() {
+  local option value default
+  option="$1"
+  default="$2"
+  value=$(tmux show-option -gqv "$option")
+
+  if [ -n "$value" ]
+  then
+    if [ "$value" = "null" ]
+    then
+      echo ""
+
+    else
+      echo "$value"
+    fi
+
+  else
+    echo "$default"
+
+  fi
+}
```

Comment:
```suggestion
get_tmux_option() {
    declare option value default
    option="$1"
    default="$2"

    value="$(tmux show-option -gv "$option")"
    if [ ! "${?}" -eq 0 ]; then
        # Option not set
        value="$default"
    fi

    echo "${value}"
}
```

This would have this as result which I personally prefer over setting null
```sh
$ tmux set -g @empty_option ""
$ tmux set -g @example_option "example"
$ tmux set -ug @does_not_exist_option
$ bash get_tmux_option.bash "@empty_option" "default"
[]
$ bash get_tmux_option.bash "@example_option" "default"
[example]
$ bash get_tmux_option.bash "@does_not_exist_option" "default"
[default]
$ cat cat get_tmux_option.bash
#!/usr/bin/env bash

get_tmux_option() {
    declare option value default
    option="$1"
    default="$2"

    value="$(tmux show-option -gv "$option")"
    if [ ! "${?}" -eq 0 ]; then
        # Option not set
        value="$default"
    fi

    echo "${value}"
}

tmp=$(get_tmux_option "$@")
echo "[$tmp]"
```

But since this is breaking change maybe a separate PR if at all

Reply:
This looks good. Next improvement that adds a breaking change should contain this also.

Reply:
Will make a PR with a new breaking change tag once this gets merged.